optuna/importance/_base.py

```diff
@@ -72,7 +72,7 @@ def _get_distributions(study: Study, params: Optional[List[str]]) -> Dict[str, B
     _check_evaluate_args(completed_trials, params)
 
     if params is None:
-        return intersection_search_space(study, ordered_dict=True)
+        return intersection_search_space(study.trials, ordered_dict=True)
```

Comment:
`study.trials` internally calls `study.get_trials` with `deepcopy=True`. This has some performance implications given that the earlier `IntersectionSearchSpace` called this method with `deepcopy=False`. Could you replace this`study.trials` with `study.get_trials(deepcopy=False)` , and similarly for other invocations?

Reply:
Thank you for your comment. I fixed this point.